README.md

```diff
@@ -19,53 +19,105 @@ how it works in [this blog post](http://blog.humphd.org/thimble-and-bramble/).
 
 Thimble requires a modern web browser, and we recommend using Mozilla Firefox or Google Chrome.
 
-Setup
------
+Thimble interacts with the Publish API (source managed in [publish.webmaker.org](https://github.com/mozilla/publish.webmaker.org)) to store users, projects, files and other content as well as publish user projects.
+
+For authentication and user management, Thimble uses Webmaker OAuth which consists of the Webmaker ID System (source managed in [id.webmaker.org](htps://github.com/mozilla/id.webmaker.org)) and the Webmaker Login API (source managed in [login.webmaker.org](https://github.com/mozilla/login.webmaker.org)).
+
+All three services along with Thimble are bundled together using Git subtrees and run together using Vagrant.
+
+Setup/Installation
+--------
 
-### Requirements
+## 1) Pre-Requisites
+In order for Thimble to be installed correctly, the following dependencies needs to be installed:
 
+- Brackets (Bramble) [[download](https://github.com/mozilla/brackets)]
```

Comment:
let's remove the "download" link and instead link it to the "Installing Brackets (Bramble)" section